### PR TITLE
smbclient: add page

### PR DESCRIPTION
**Pull request by MrLeebo**
----

- [x] The page (if new), does not already exist in the repo.

- [x] The page (if new), has been added to the correct platform folder:  
      `common/` if it's common to all platforms, `linux/` if it's Linux-specific, and so on.

- [x] The page has 8 or fewer examples.

- [x] The PR is appropriately titled:  
      `<command name>: add page` for new pages, or `<command name>: <description of changes>` for pages being edited

- [x] The page follows the [contributing](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) guidelines

### CODE REVIEW

File: pages/linux/smbclient.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+> FTP-like client to access SMB/CIFS resources on servers.
+
+- Connect to a share (`exit` to quit the session):
+
+`smbclient {{//server/share}}`
+
+- Connect with a username and password:
+
+`smbclient {{//server/share}} --user {{username%password}}`
```
Comment (agnivade) [inline pages/linux/smbclient.md]: Is the % required ? Then it should be {{username}}%{{password}}.

File: pages/linux/smbclient.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
+# smbclient
+
+> FTP-like client to access SMB/CIFS resources on servers.
+
+- Connect to a share (`exit` to quit the session):
```
Comment (agnivade) [inline pages/linux/smbclient.md]: By default, will it prompt for a username/password ?

Comment (MrLeebo) [reply pages/linux/smbclient.md]: It will prompt for password, default username will be your current user.

File: pages/linux/smbclient.md (hunk 1 above)
Comment (MrLeebo) [reply pages/linux/smbclient.md]: I wasn't sure how to display that... Technically `%password` is optional if you want to enter username only and use the prompt to enter the password. `{{username}}{{%password}}`?

Comment (agnivade) [reply pages/linux/smbclient.md]: You can split it into 2 examples. One having {{username}}, another having {{username%password}}.

File: pages/linux/smbclient.md (hunk 2 above)
Comment (agnivade) [reply pages/linux/smbclient.md]: You can specify that within the brackets.

**Pull request merged**